luaui/Widgets/gui_comselect.lua

```diff
@@ -0,0 +1,88 @@
+function widget:GetInfo()
+	return {
+		name = "Commander Selector",
+		desc = "Allows for proper keybind selection of any flavour of commanders",
+		author = "Hornet, with some ZK raiding",
+		date = "03 June 2024",
+		license = "GNU GPL, v2 or later",
+		layer = 1001,
+		enabled = true,
+	}
+end
+
+
+
+local myTeamID = Spring.GetMyTeamID()
+
+local commanderList = {}
+local commanderIds = {}
+for udid, ud in pairs(UnitDefs) do
+	if ud.customParams and ud.customParams.iscommander then
+		commanderIds[udid] = udid
+	end
+end
+
+
+
+function widget:UnitCreated(unitID, unitDefID, unitTeam)
```

Comment:
missing UnitDestroyed. better yet though, use MetaUnitAdded/Removed

Reply:
I had it before, removed it while debugging things, will recode.

Reply:
@sprunk is there any documentation for `MetaUnitAdded`? Its purpose isn't clear to me, and its only use in BAR currently is some debug code.